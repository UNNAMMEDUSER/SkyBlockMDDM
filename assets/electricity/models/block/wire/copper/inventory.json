{
	"parent": "general:item/resource/copper/wire"
}