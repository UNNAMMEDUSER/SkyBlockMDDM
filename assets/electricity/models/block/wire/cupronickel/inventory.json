{
	"parent": "general:item/resource/cupronickel/wire"
}