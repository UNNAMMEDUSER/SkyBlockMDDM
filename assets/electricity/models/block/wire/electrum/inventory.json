{
	"parent": "general:item/resource/electrum/wire"
}