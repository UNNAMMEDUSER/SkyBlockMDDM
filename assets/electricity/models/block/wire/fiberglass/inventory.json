{
	"parent": "general:item/resource/fiberglass/wire"
}