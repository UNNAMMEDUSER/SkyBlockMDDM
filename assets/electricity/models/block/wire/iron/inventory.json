{
	"parent": "general:item/resource/iron/wire"
}