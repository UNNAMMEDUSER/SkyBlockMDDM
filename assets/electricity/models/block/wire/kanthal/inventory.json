{
	"parent": "general:item/resource/kanthal/wire"
}