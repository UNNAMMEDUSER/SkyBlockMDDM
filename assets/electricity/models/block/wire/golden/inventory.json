{
	"parent": "general:item/resource/golden/wire"
}